tests/regression/src/lib.rs

```diff
@@ -144,6 +146,12 @@ mod tests {
             format!("target/{}/{}.raw", self.commit_hash, self.test_name)
         }
 
+        // Returns the annotated profile asociated with a raw profile
```

Comment:
```suggestion
        // Returns the annotated profile associated with a raw profile
```
Can you turn on a spell checker in your IDE? Also in general can you spellcheck this crate? It looks like there are a lot of spelling mistakes in the README. It's fine to do that in a different PR if you want.

Reply:
Yep, I'll do that in this PR